notebook_utils/berdl_notebook_utils/berdl_settings.py

```diff
@@ -24,8 +24,8 @@ class BERDLSettings(BaseSettings):
 
     # MinIO configuration
     MINIO_ENDPOINT: str = Field(..., description="MinIO endpoint (hostname:port)")
-    MINIO_ACCESS_KEY: str
-    MINIO_SECRET_KEY: str
+    MINIO_ACCESS_KEY: str = Field(default="access_key_placeholder", description="MinIO access key")
```

Comment:
I think we should not have defaults were possible

Reply:
The MINIO_ACCESS_KEY and MINIO_SECRET_KEY values are unusual. I need to pass BERDLSettings to the MinIO client to retrieve the user’s secret key, but at the time BERDLSettings is provided, both the access key and secret key are None. I am planning to remove the hub hook and use the MinIO service to obtain the user’s access and secret keys instead in the user's notebook.